src/routers/authRouter.js

```diff
@@ -0,0 +1,46 @@
+// src/routers/authRouter.js
+const { Router } = require('express');
+const { 
+    signup, login, refreshToken, 
+    getUserProfile, updateUserProfile, changePassword, 
+    getUserProducts, getLikedProducts 
+} = require('../controllers/authController');
+const { authenticateUser } = require('../lib/auth/middleware');
+const { uploadImage } = require('../src/middlewares/index'); // Multer 설정을 가져옵니다.
```

Comment:
```suggestion
const { uploadImage } = require('../lib/...'); // Multer 설정을 가져옵니다.
```
작성하신 경로가 존재하지 않습니다. 위 경로가 맞는 거 같아요.